edx_exams/apps/core/management/commands/test/test_bulk_add_course_staff.py

```diff
@@ -130,3 +130,32 @@ def test_dupe_user_csv(self):
             call_command(self.command, f'--csv_path={csv.name}')
             self._assert_user_and_role(username, email, self.course_role, self.course_id)
             self._assert_user_and_role(username, email, self.course_role, course_id_2)
+
```

Comment:
Could you add a test case where you set the batch size to 2 and create like 3 course_staff users?

Reply:
Adding two cases, one where the batch size is larger than the number of roles, and one where the batch size is smaller.